docs/code/getting-started.md

```diff
@@ -319,96 +316,60 @@ Now that we've got all these swanky files laying around, we need a way to tell t
+The next thing we need is a build system. The build system that we're going to be using is called [Meson](http://mesonbuild.com/). We already installed the `meson` program at the beginning of this book when we installed `elementary-sdk`. What we're going to do in this step is create the files that tell Meson how to install your program. This includes all the rules for building your source code as well as correctly installing your .desktop and appdata files and the binary app that results from the build process.
 
-        add_definitions(${DEPS_CFLAGS})
-        link_libraries(${DEPS_LIBRARIES})
-        link_directories(${DEPS_LIBRARY_DIRS})
+Create a new file in your project's root folder called "meson.build". We've included some comments along the way to explain what each section does. You don't have to copy those, but type the rest into that file:
 
-        # make sure we have vala
-        find_package(Vala REQUIRED)
-        # make sure we use vala
-        include(ValaVersion)
-        # make sure it's the desired version of vala
-        ensure_vala_version("0.16" MINIMUM)
+        # project name and programming language
+        project('com.github.yourusername.yourrepositoryname', 'vala', 'c')
 
-        # files we want to compile
-        include(ValaPrecompile)
-        vala_precompile(VALA_C ${EXEC_NAME}
-            src/hello-again.vala
-
-        # tell what libraries to use when compiling
-        PACKAGES
-            gtk+-3.0
+        # Create a new executable, list the files we want to compile, list the dependencies we need, and install
+        executable(
+            meson.project_name(),
+            'src/Application.vala',
+            dependencies: [
+                dependency('gtk+-3.0')
+            ],
+            install: true
         )
 
-        # tell cmake what to call the executable we just made
-        add_executable(${EXEC_NAME} ${VALA_C})
-
-        # install the binaries we just made
-        install (TARGETS ${EXEC_NAME} RUNTIME DESTINATION bin)
+        #Install our .desktop file so the Applications Menu will see it
+        install_data(
+            join_paths('data', meson.project_name() + '.desktop'),
+            install_dir : join_paths(get_option('datadir'), 'applications')
+        )
 
-        # install our .desktop file so the Applications Menu will see it
-        install (FILES ${CMAKE_CURRENT_SOURCE_DIR}/data/com.github.yourusername.yourrepositoryname.desktop DESTINATION ${DATADIR}/applications/)
-        
-        # install our .appdata.xml file so AppCenter will see it
-        install (FILES ${CMAKE_CURRENT_SOURCE_DIR}/data/com.github.yourusername.yourrepositoryname.appdata.xml DESTINATION ${DATADIR}/metainfo/)
+        #Install our .appdata.xml file so AppCenter will see it
+        install_data(
+            join_paths('data', meson.project_name() + '.appdata.xml'),
+            install_dir: join_paths(get_option('datadir'), 'metainfo')
```

Comment:
I am sorry but it appeared that I didn't take the meaning of your words correctly...

I checked the changes, and this does not cause errors, but maybe it is better to insert a space after `install_dir`.
That is, `install_dir : join_paths(get_option('datadir'), 'metainfo')`.

Reply:
I found this comment was wrong. I'm sorry.
Instead, please see https://github.com/elementary/website/pull/1937#discussion_r198079642